README.md

```diff
@@ -0,0 +1,34 @@
+# Ethereum APIs
```

Comment:
Change the title of this to eth2? Shall we rename to repo to eth2-apis? Context is important here about what specifically this targets

Reply:
Hmm, I am up for debating this and I'm probably in the minority here but this is my thinking:

Ethereum 2.0 is _Ethereum_. It's a (major) upgrade to the network but it is still fundamentally Ethereum. If we continue to label all of Ethereum as Eth# then it might be confusing down the line. While ETH2 is a radical change, imagine that ETH3 isn't a radical change but users are confused about the pattern we've set by labeling ETH3 as a breaking change or distinct new project. 

I think we leave it as Ethereum and make this the general API repo for Ethereum core APIs

Reply:
Also, there is no reason this couldn't also contain ETH1 schemas as well, if that were possible.

Reply:
LOL semantics.. love you guys